core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -52,16 +52,27 @@ case object Paused extends ContainerState
 case object Removing extends ContainerState
 
 // Data
-sealed abstract class ContainerData(val lastUsed: Instant, val memoryLimit: ByteSize)
+sealed abstract class ContainerData(val lastUsed: Instant,
+                                    val memoryLimit: ByteSize,
+                                    val activeActivationCount: Int = 0)
 case class NoData() extends ContainerData(Instant.EPOCH, 0.B)
 case class MemoryData(override val memoryLimit: ByteSize) extends ContainerData(Instant.EPOCH, memoryLimit)
-case class PreWarmedData(container: Container, kind: String, override val memoryLimit: ByteSize)
-    extends ContainerData(Instant.EPOCH, memoryLimit)
+case class PreWarmedData(container: Container,
+                         kind: String,
+                         override val memoryLimit: ByteSize,
+                         override val activeActivationCount: Int = 0)
```

Comment:
Agreed, will work on that; 
PreWarmedData may be 1 when emitted during cold start, to signal to parent that it is prewarmed, but already has active activations in flight, so that pool can use it asap, but also respect the concurrency limit.